DESCRIPTION

```diff
@@ -31,8 +31,9 @@ Imports:
 Suggests: 
     rmarkdown,
     testthat (>= 3.0.0),
+    ifadv,
     pkgdown,
     viridis,
```

Comment:
viridis is not used in the workflow. It should be removed from the description.

Reply:
Issue created